### ISSUE TITLE
Dynamic class map to import different CSV files

### ISSUE THREAD

**Issue by kush316** [JoshClose/CsvHelper#1841]
Hello,

I have a requirement where I am building an utility to import data from CSV to SharePoint and we are receiving different file from different LOB applications. 

I would like to have mapping class generated dynamically so that I can use same utility for data import.

Can't have individual application as could be receiving 100s of files with different schema for import.

Is it possible using CSVHelper?

**Comment by JoshClose** [comment]
Yes you can create a map on the fly and add it. It may just be easier to read/write by hand like `csv.GetField<int>("my column")`.
